main.cpp

```diff
@@ -110,17 +110,17 @@ Midi Controller
  */
 
 /*
-Thing 10) House
+Thing 10) Car
 5 properties:
-    1) Number of Doors
-    2) Type of Roof
-    3) Color of Paint
-    4) Size of Windows
-    5) Acreage
+    1) Brand of Radio
+    2) Number of Tires
+    3) Size of Engine
+    4) Brand of Rims
+    5) Count of Seats
```

Comment:
These are individual properties, not sub-objects
`number of tires` can be represented as an `int`:  `int numTires = 5; //includes spare`
`Brand of Rims` can be represented with a string: `std::string rimBrand = "BMW";`
- [ ] revise